TERN.md

```diff
@@ -68,8 +76,11 @@ We intend to enhance TURTLEDOVE in the following privacy-preserving ways:
 - Better supporting dynamic creative and product recommendation use cases
 - Supporting some functionality of third-party tags
 - Further specifying auditability of delivered ad creatives
+- Enabling brand safety for both advertisers and publishers
+- Creating a mechanism for trackability metrics
 - Adding a `privateData` object that can never escape the browser, but improve bidding signals (among potential other use cases)
-- Specifying the auction dynamic to avoid game-theoretic complications
+- Creating a mechanism to create interest groups based on publisher browsing behavior
```

Comment:
(This line should go away also, right?)

Reply:
Good catch; didn't come up when i was searching for "contextual." It's been removed.